pkgs/top-level/python-packages.nix

```diff
@@ -6226,8 +6234,6 @@ in {
 
   pyro-ppl = callPackage ../development/python-modules/pyro-ppl { };
 
-  pyroon = callPackage ../development/python-modules/pyroon { };
```

Comment:
This should get an alias, too.

Reply:
I disagree. It was introduced on 2021-05-28 into master and home-assistant is its only consumer within nixpkgs. I think it is fair to say we can deprecate the wrong name with impunity.